### ISSUE TITLE
Feature request: Ability to tag multiple selections at once

### ISSUE THREAD

**Issue by mdalch** [elementary/photos#511]
It would be much easier to manage a photo library when there is the ability to edit metadata of multiple photos at once.

**Comment by jeremypw** [comment]
Which metadata do you think should be applied to multiple photos? You can already change the date and time. It does not make much sense to apply e.g. exposure time.  Applying tags to multiple photos is not obvious but can be done by  drag'n'drop.

**Comment by mdalch** [comment]
I didn't know the drag'n'drop feature. this is cool. But this works only for adding Tags. How about when someone have done a mistake and wants to edit the Tag of those photos again, in this scenario the editing must be done photo by photo. Because while selecting multiple photos the right side bar disappears (the side bar contains a field to write a tag). So there is no way to edit Tags in such a scenario.

**Comment by jeremypw** [comment]
Thats true. All you can do at the moment is to delete the tag, recreate it and add the correct selection - not ideal!

Ideally the sidebar should show common properties of the selection and allow editing them.
